tools/convert_to_onnx.py

```diff
@@ -98,7 +98,8 @@ def main():
                                     num_classes=args.num_classes, 
                                     snap_path=cfg.model.load_weights)
     model = build_model(**model_kwargs(cfg, num_classes))
-    load_pretrained_weights(model, cfg.model.load_weights)
+    if cfg.model.load_weights:
```

Comment:
There is no use in skipping the weights loading step silently, excepting debugging. At least we need a clearly visible warning here.